### ISSUE TITLE
[Question] Witch Tower block of netherite

### ISSUE THREAD

**Issue by polykatt** [SuperMartijn642/FormationsOverworld#8]
Hello, I was wondering about this loot found in the witch tower. These structures are really overpowered, especially since if you find one even early game its really easy to loot. You'll get easy access to enchanting immediately, plus strong enchanted books found in the chiseled bookshelves. And a block of netherite, which is enough for full armor and tools - just seems way too overpowered and easy. The structure is cool with good loot but maybe it's a bit over the top; maybe place another diamond block in its place? Netherite block in overworld structure is just not suited I think.

**Comment by SuperMartijn642** [comment]
Other people have brought this up as well and I will go over all the structures in the mod and replace all the overpowered loot at some point.
I don't really play the game myself, so judging what to put in structures in somewhat difficult. For example, I simply didn't realize how difficult netherite was to obtain, I thought it was much more plentiful.

> strong enchanted books found in the chiseled bookshelves

Haven't heard anything about enchanted books. I can't remember exactly how it works, but it is some processor which randomly generates enchanted books in chiseled bookshelves. As far as I remember I just set some values and then visited some towers to see what was there and it seemed okay.

Will probably be a bit before I get to working on the Formations mods again as I have quite a lot of issues and mods I want to work on 😅

**Comment by polykatt** [comment]
Yeah, netherite is pretty rare and only found in the nether. I'd say even diamonds and some other ores are made too easy with a few of the structures in this mod. This is a cool mod though, looking forward to updates when you get to it!

**Comment by BobzBlue** [comment]
Simply having the enchanting table in the stone tower is a bit much... even the blast furnace or anvil as they use a lot of iron. A tool or weapon smith villager house might have a diamond, but an enchanted diamond shovel goes way beyond that.

**Comment by TheBedrockMaster** [comment]
This would be great.

**Comment by VaOnechezz** [comment]
@SuperMartijn642 I wanna mention this problem again.
I love most of structures from this mod but this witch_tower completely ruins balance.
I haven't visited nether yet, I haven't even got enough diamonds for items and armor
But this witch_tower....
It gives diamonds, **FULL netherite block**. There's also laod of books, enchantment table, lapis blocks, golden blocks, brewery stand and things for this etc. It's like u almost don't need to go digging, u don't need to go nether since u have everything here. 
![Image](https://github.com/user-attachments/assets/8493d826-5269-4c45-b325-c515dc321052)

Maybe I would check some other structures if they're balanced 👀 
And one more thing.. I use "sparse structures" mod that allows to change how often or rare structures are located, it also gives opportunity to configure this value for exact sets
This is example of how I configured some other things (and also global spreadFactor is 1.7)
![Image](https://github.com/user-attachments/assets/b8496683-518a-4d3d-bfc8-e46f1024b703)

But I understood that I can't configure same way with formationsOverworld since it gives only 3 sets, so if I wanna delete/make witch_tower way more rare - I can't impact exactly this structure, I will also hit large_temple as it's being part of set where witch_tower belongs to. 
It looks like I will need to learn something about datapacks to disable this structure..
![Image](https://github.com/user-attachments/assets/d2501efa-a000-4f0e-8414-790298f2c6e3)

But overall I like most of structures from this mod!

**Comment by VaOnechezz** [comment]
~~
I just turned off formationsoverworld:rare set by sparsestructures mod

**Comment by SuperMartijn642** [comment]
I have removed the overpowered loot mentioned here from the witch tower and I have also adjusted all loot tables throughout the mod. Hopefully the structures should be more balanced now.
These changes are available in Formations Overworld 1.0.5.
Thank you all for the feedback and suggestion, and feel free to let me know if things still seem out of place and could be balanced better!
